backend/app.js

```diff
@@ -123,25 +126,9 @@ users_db.defaults(
     }
 ).write();
 
-// config values
-var frontendUrl = null;
-var backendUrl = null;
-var backendPort = null;
-var usingEncryption = null;
-var basePath = null;
-var audioFolderPath = null;
-var videoFolderPath = null;
-var downloadOnlyMode = null;
-var useDefaultDownloadingAgent = null;
-var customDownloadingAgent = null;
-var allowSubscriptions = null;
-var subscriptionsCheckInterval = null;
-var archivePath = path.join(__dirname, 'appdata', 'archives');
 
-// other needed values
-var options = null; // encryption options
-var url_domain = null;
 var updaterStatus = null;
+var archivePath = path.join(__dirname, 'appdata', 'archives');
```

Comment:
This should be refactored

Reply:
It can be part of config, it should ideally be configurable anyways.